src/develop/imageop.c

```diff
@@ -549,10 +549,13 @@ static void dt_iop_gui_delete_callback(GtkButton *button, dt_iop_module_t *modul
   dt_dev_module_remove(dev, module);
 
   // we recreate the pipe
+/*
+  // pipe will be recreated below
```

Comment:
I don't see where. It is refreshed later in the routine. In any case I would certainly remove those commented out code if it is not necessary. And add a hint where the pipe is recreated.

Reply:
I left this just for the review, I'll remove it later.
Where it says refreshed, the effect is the same.

Reply:
Understood. Then better moving the message indeed.